indico/modules/events/abstracts/templates/abstract.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'events/abstracts/management/_base.html' %}
+
+{% from 'events/abstracts/_abstract/public.html' import render_abstract_public, render_abstract_comments, render_edited_hint %}
+{% from 'events/abstracts/_abstract/judge.html' import render_decision_box %}
+{% from 'events/abstracts/_abstract/review.html' import render_abstract_reviews %}
+
+{% block content %}
+    {{ render_abstract_public(abstract, management=management) }}
+    <div id="abstract-private" class="i-timeline">
+        <div class="i-timetable-connect-down"></div>
+        <div class="i-timeline-separator"></div>
+        {{ render_abstract_comments(abstract) }}
```

Comment:
We are importing `render_abstract_comments` from `public.html` but it's clearly something strictly non-public. I would move it somewhere else. Maybe `review.html`?